code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -311,6 +311,30 @@
 	for(var/Z in limb_list)
 		. += regenerate_limb(Z, noheal)
 
+/mob/living/proc/regenerate_organs()
```

Comment:
put it in the organ file, not the limbs file, two different concepts these days.